pkg/odo/cli/pipelines/bootstrap.go

```diff
@@ -82,9 +82,9 @@ func NewCmdBootstrap(name, fullName string) *cobra.Command {
 
 	initCmd := &cobra.Command{
 		Use:     name,
-		Short:   bootstrapShortDesc,
-		Long:    bootstrapLongDesc,
-		Example: fmt.Sprintf(bootstrapExample, fullName),
+		Short:   initShortDesc,
```

Comment:
We don't have to change these to `init`. I guess it happened while copying files

Reply:
yes I have fixed it.. Pushing it in next commit